### DISCUSSION TITLE
Kimai Demo - enable CORS (Access-Control-Allow-Origin)

### DISCUSSION THREAD

**Discussion by KatjaGlassConsulting** [kimai/kimai#4127] [Questions & Answers]
Hi Kevin,

would it be possible to set the "Access-Control-Allow-Origin: *" on the demo server? 

Background: I would like to use your demonstration server for live demonstration of API utilization. After updating TimeKex (Web Application connecting to Kimai via API - also on the Marketplace), it is now working with the latest Kimai2 Version 2, but I get now CORS errors. 

Testing: When you try the app here (https://katjaglassconsulting.github.io/TimeKex/app/#/) and login as John_user with the API password, the "Kimai General Data Status" should be green, not red.

Alternative: If you don't like to open it up for all, maybe you could allow this URL as origin?

Thanks,
Katja

**Comment by kevinpapst** [comment]
I don't know where the issue happens. Tried that in `.env` but still not working:
```
CORS_ALLOW_ORIGIN=^https?://katjaglassconsulting.github.io?$
```

Maybe a webserver configuration, hard to say without time for real debugging.
